genologics/descriptors.py

```diff
@@ -542,6 +542,20 @@ def __get__(self, instance, cls):
                 pass
         return self.value
 
+    def __set__(self, instance, values):
+        instance.get()
+        nodes = instance.root.findall('reagent-label')
+        if not nodes:
+            root = instance.root
+            for value in values:
+                rl_node = ElementTree.SubElement(root, 'reagent-label', name=value)
```

Comment:
rl_node is not used anywhere else. Is it a mistake? If it has to be there, I think 'dummy' is better.

Reply:
Can run without it, I'll nuke it

Reply:
Done